application/di/test_init.go

```diff
@@ -60,15 +60,15 @@ func TestInit(t *testing.T) {
 	authProvider := snyk.NewFakeCliAuthenticationProvider()
 	snykApiClient = &snyk_api.FakeApiClient{CodeEnabled: true}
 	authenticationService = snyk.NewAuthenticationService(authProvider, analytics, errorReporter, notifier)
-	cliInitializer = cli.NewInitializer(errorReporter, installer, notifier)
+	cliInitializer = cli.NewInitializer(errorReporter, installer, notifier, nil)
```

Comment:
shouldn't the snykCli assignment be moved above this line then passed  to the initializer instead of nil?

Reply:
Fixed